packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -325,7 +325,33 @@ class ErrorSpacer extends DiagnosticsProperty<void> {
 
 /// Class for information provided to [FlutterExceptionHandler] callbacks.
 ///
-/// See [FlutterError.onError].
+/// {@tool snippet}
+/// This is an example of using [FlutterErrorDetails] when calling
+/// [FlutterError.reportError].
+///
+/// ```dart
+/// bool _runningAsyncTasks = true;
```

Comment:
we can turn this into a real sample with ```{@tool sample --template=...}```

Reply:
why do we want to include this boolean in this sample?

Reply:
+1 to simplifying this example slightly be removing this variable and also the "return" keywords. Just a try/catch with the error report in the catch block should be enough.

Reply:
I simplified the sample. @chunhtai do you mean to make it a sample app that crashes?